src/pages/community-champions.js

```diff
@@ -1,8 +1,73 @@
 import React from 'react';
+import styled from '@emotion/styled';
 import Layout from '~components/dev-hub/layout.js';
+import HeroBanner from '~components/dev-hub/hero-banner';
+import { P, H2 } from '~components/dev-hub/text';
+import Button from '~components/dev-hub/button';
+import { screenSize, size } from '~components/dev-hub/theme';
+import BannerImage from '~images/community-champions/community-champions-banner-icon.png';
+
+const APPLY_BUTTON_BOTTOM_MARGIN = '14px';
+const APPLY_BUTTON_MOBILE_BOTTOM_MARGIN = '12px';
+
+const Subtitle = styled(P)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    margin-top: ${size.mediumLarge};
+    @media ${screenSize.upToLarge} {
+        margin-top: ${size.default};
+    }
+`;
+
+const ApplyButton = styled(Button)`
+    margin-top: ${size.large};
+    margin-bottom: ${APPLY_BUTTON_BOTTOM_MARGIN};
+    @media ${screenSize.upToLarge} {
+        margin-top: ${size.default};
+        margin-bottom: ${APPLY_BUTTON_MOBILE_BOTTOM_MARGIN};
+    }
+`;
+
+const StyledHeroBanner = styled(HeroBanner)`
+    > div {
```

Comment:
This is great since we are able to extend the styling for the banner, could we leave a comment here actually explaining what the CSS is doing? This is just to help us understand what the goal is for clarity.